styles/layout.css

```diff
@@ -0,0 +1,58 @@
+label {
+    float: left;
+    margin: 20px;
+}
+
+textarea {
+    display: block;
+    float: left;
+}
+
+#make {
+    float: right;
+    max-width: 45%;
+}
+
+#prev {
+    float: left;
+    max-width: 45%;
+    word-break: keep-all;
+    border: 2px solid black;
+    padding: 2%;
+    word-break: break-all;
```

Comment:
You've got `word-break` in here twice. CSS will select the last property as it cascades.